src/MessageHandler/ActivityPub/Inbox/ActivityHandler.php

```diff
@@ -101,6 +101,16 @@ private function handle(?array $payload)
             }
             if (\is_array($payload['object'])) {
                 $payload = $payload['object'];
+                $actor = $payload['actor'] ?? $payload['attributedTo'] ?? null;
+                if ($actor) {
+                    $user = $this->manager->findActorOrCreate($actor);
+                    if ($user instanceof User && null === $user->apId) {
```

Comment:
What about other local announcements..?

Reply:
What do you mean?

Reply:
I mean, could there be local magazines, posts, ... also triggering "Announce" messages?

Reply:
This only covers incoming announces. It is in the activity handler, so it is **the** poijt of incoming activites from our inbox. 

However I just realised that it will probably break boosts, as they are annpjnce activites as well. I think I already broke boosts in some earlier change,but I'll have to investigate this

Reply:
Ok, so I checked, I broke boosting with
- the `AnnounceWrapper` changes in c13bf815
- and the `Outbox/AnnounceHandler` changes in 9eddb84f

So I think we can just ignore that part and I'll make another issue about it

Reply:
OK hopefully you could indeed fix the boost regression soon 👍🏽 .